flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBResourceContainer.java

```diff
@@ -207,4 +221,37 @@ public void close() throws Exception {
             sharedResources.close();
         }
     }
+
+    /**
+     * Overwrite configured {@link Filter} if enable partitioned filter. Partitioned filter only
+     * worked in full bloom filter, not blocked based.
```

Comment:
```suggestion
     * Overwrite configured {@link Filter} when partitioned filter is enabled, since partitioned filter only
     * works with full bloom filter, not blocked based.
```